Normalizer/CustomEntityNormalizer.php

```diff
@@ -43,21 +43,16 @@ public function normalize($entity, $format = null, array $context = []): array
     {
         $normalizedEntity = $this->pimSerializer->normalize($entity, 'standard', $context);
 
-        $meta = [
-            // Seems the upstream pim introduced some versioning for the javascript assets.
-            // Not sure how to implement this correct and dont have the time to investigate this now.
```

Comment:
Actually the `structure_version` key is related to the entity's version (and not to the js asset's one)